pkg/integration/tests/commit/commit_multiline.go

```diff
@@ -22,7 +22,16 @@ var CommitMultiline = NewIntegrationTest(NewIntegrationTestArgs{
 			PressPrimaryAction().
 			Press(keys.Files.CommitChanges)
 
-		t.ExpectPopup().CommitMessagePanel().Type("first line").AddNewline().AddNewline().Type("third line").Confirm()
+		t.ExpectPopup().CommitMessagePanel().
```

Comment:
We wanna have a more ergonomic way of doing this stuff. Something like:
```go
t.ExpectPopup().CommitSummaryPanel().
	Type("first line").
	SwitchToDescription().
	AddNewline().
	AddNewline().
	Type("third line").
	SwitchToSummary().
	Confirm()
```

We should be able to get this behaviour by creating a new driver component i.e. a `CommitDescriptionPanelDriver`, and having methods to switch between the two panels